tests/integration_test.rs

```diff
@@ -267,9 +267,9 @@ async fn test_policy_evaluator(
     "app_deployment.json",
     rego_scenario
 )]
-#[case::wapc_cani(
+#[case::wapc_can_i(
     PolicyExecutionMode::KubewardenWapc,
-    "ghcr.io/kubewarden/tests/context-aware-test-policy:v0.2.0",
+    "ghcr.io/jvanz/tests/context-aware-test-policy:v0.2.1",
```

Comment:
This should be updated when changes from https://github.com/kubewarden/context-aware-test-policy/pull/63 is released